python/lsst/ts/observing/utilities/auxtel/latiss/utils.py

```diff
@@ -15,6 +29,28 @@ def parse_obs_id(obs_id):
     return source, controller, dayObs, seqNum
 
 
+def parse_visit_id(visit_id):
+    """Return a data_id dictionary from a visit ID.
+
+    Parameters
+    ----------
+    visit_id: `int` or `str`
+        Visit id (e.g. '2021032300308')
+
+    Returns
+    -------
+    data_id: `dict`
+        dictionary with dayObs and seqNum keys
+    """
+    _visit_id = str(visit_id)
+    day_obs = f"{_visit_id[0:4]}-{_visit_id[4:6]}-{_visit_id[6:8]}"
+    seq_num = int(_visit_id[9::])
+
+    data_id = dict(dayObs=day_obs, seqNum=seq_num)
+
+    return data_id
```

Comment:
I wish parse_obs_id also returned a dict like this, or a dataclass. But I suppose changing that would be too much work?

Dataclasses are a bit nicer than dicts (and a _lot_ nicer than tuples) because you have attribute access and a place to document the contents and field types.

Reply:
I agree.
At the moment it'd be a pain, however, we're going to re-write the CWFS script once the AOS is done and that will be the time to do it.
I'd also need the NTS to be up in order to really test it...  so we'll leave this for another day.